examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST (works only if running on localhost)</h4>
+<form method="post"
+      action-xhr="http://iframe.localhost:8000/form/redirect-to/post"
+      target="_blank">
```

Comment:
> Supporting _blank would require the page to opt into the redirect with a new attribute like will-redirect.

Would it be possible to open a new window given the async nature of the submission?

> At minimum we should do something about the code above. target=_blank but redirect happening on top does not make sense. Can we change validation rule so either (target | action-xhr) is required?
  
Yes we discussed un-requiring `target` for XHR cases and we have an issue to follow up on #5607

The requirement was the initial thought but I think we're open to dropping it.